src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.interfaces.AreaCalculable;
+import core.basesyntax.interfaces.Drawable;
+
+public abstract class Figure implements Drawable, AreaCalculable {
+
+    protected String color;
```

Comment:
Consider using an enum for the color property instead of a String to ensure that only predefined colors can be used for figures.